test/common/router/config_impl_test.cc

```diff
@@ -2128,6 +2132,40 @@ TEST(RouteMatcherTest, TestDuplicateDomainConfig) {
                EnvoyException);
 }
 
+TEST(RouteMatcherTest, TestCaseSensitiveDomainConfig) {
```

Comment:
For new tests please proceed the test with a single line `//` comment talking about what the test does.

Reply:
Sure. Will add it.